plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -0,0 +1,59 @@
+package io.jenkins.tools.pluginmanager.cli;
+
+import io.jenkins.tools.pluginmanager.config.Settings;
+import java.io.File;
+import javax.annotation.Nonnull;
+import org.kohsuke.args4j.Option;
+import org.kohsuke.args4j.spi.BooleanOptionHandler;
+import org.kohsuke.args4j.spi.StringArrayOptionHandler;
+
+
+
+public class CliOptions {
+    //path must include plugins.txt
+    @Option(name = "-pluginTxtPath", usage = "Path to plugins.txt")
+    public File pluginTxt;
+
+    @Option(name = "-pluginDirPath", usage = "Directory in which to install plugins")
+    public File pluginDir;
+
+    @Option(name = "-plugins", usage = "List of plugins to install, separated by a space", handler = StringArrayOptionHandler.class)
+    public String[] plugins;
+
+    @Option(name = "-war", usage = "Path to Jenkins war file")
```

Comment:
What is the use-case for this parameter? It is not used ATM

Reply:
My understanding of the original script is that the bundledPlugins checks the war file in /usr/share/jenkins/jenkins.war for plugins.  Will this war file always be in the same location?